serverless/cmd/feeder/main.go

```diff
@@ -0,0 +1,185 @@
+// Copyright 2021 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// feeder is a witness feeder implementation for the serverless log.
```

Comment:
This doc would be better saying that it's a one-shot tool to feed a single checkpoint. I would otherwise expect a "witness feeder implementation" to be a long lived process that polls a log.

Reply:
done - this was a funny one; it was "obvious" (:D) to me that this ("of course") is a one-shot, since it'll be run in the same action as whatever runs the `integrate` command, conveniently forgetting that someone might want to have a long-running feeder for serverless logs :)

ta